config_templates/gretel/synthetics/dgan.yml

```diff
@@ -0,0 +1,26 @@
+# Default configuration for specialized time series model.
+
+schema_version: 1.0
+name: "dgan-default-configuration"
+models:
+  - timeseries_dgan:
+        data_source: "_"
+
+        # Replace with the column name containing the date or time
+        # values in your dataset.
+        time_column: "Date"
+        df_style: "long"
+        params:
+            sample_len: 1
+            max_sequence_len: 10
```

Comment:
If this (or others) variable is very likely to need to change for a user's dataset, then I think we should have a comment about it the same way we do for the time_column.